### PR TITLE
Configure devcontainer.json with hugomods/hugo

### PR DESCRIPTION
**Pull request by louwers**
Use image with more up-to-date Hugo version than default image for dev container.

https://docker.hugomods.com/

### CODE REVIEW

File: .devcontainer/devcontainer.json
Comment (HarelM) [inline .devcontainer/devcontainer.json:2]: That would fail, you'd want 0.113.0 I believe.

**Comment by HarelM** [comment]
Are you able to run a docker command to serve the hugo site using docker?

**Comment by HarelM** [comment]
Can you update the readme to allow running the site with only docker installed?

**Comment by louwers** [comment]
Yes it works.

**Review by HarelM — APPROVED** [review]
Generally approved although I would still recommend improving the readme on how to use docker instead of the need to install locally the relevant version.

**Comment by louwers** [comment]
That's a separate issue I think.

**Comment by louwers** [comment]
I can add a Dockerfile.

**Comment by HarelM** [comment]
I was not able to make it work using the docker command in the documentation, so I'm less optimistic than you are I guess...
Your call, I have little patience to this repo and the way things are maintained here...

**Comment by HarelM** [comment]
I don't think a dockerfile is needed as you don't need to publish a docker image, just run the docker locally to avoid the need to install hugo.
A simple docket run --rm -v... Should do, but it didn't work for me, so IDK...

**Comment by HarelM** [comment]
I'll merge this, at least see if I can make the site run...

**Pull request merged**
